### ISSUE TITLE
Weights support missing

### ISSUE THREAD

**Issue by SirJohnFranklin** [keras-team/autokeras#1372]
Dear authors of autokeras,

do you plan to include sample wise weighting for the loss function like it is implemented in keras?

**Comment by haifeng-jin** [comment]
With [ClassificationHead](https://autokeras.com/block/#classificationhead), you can use a custom loss function. Please let us know if it works for you. You need to use the AutoModel API with it.
